.github/workflows/deploy-docs.yml

```diff
@@ -3,7 +3,7 @@ name: Deploy Docs
 on:
   push:
     branches:
-      - main
```

Comment:
This should not have to change as PRs automatically get deploy previews

Reply:
using for testing only - for some reason the theme configuration is not working properly when switching from light to dark mode, both images are displayed (see page 7) locally it works as expected.

Reply:
https://github.com/mitre/saf-training/pull/304

Was told to share this with you.  Upstream continues to make breaking changes.  You might want to see if changing where the setting was placed fixes your issue that you're having.